content/5_Plat/Sweep_Line.mdx

```diff
@@ -46,7 +46,7 @@ Then $\delta$ is the upperbound of the answer. If a more optimal answer exists,
 
 A brute force matching algorithm that computes $\texttt{dist}(p, q)$ for all $p \in L'$, $q \in R'$ would have a worst-case runtime of $(\frac n 2)^2 = \mathcal{O}(n^2)$ (recall that $L'$ and $R'$ may have up to $\frac N 2$ points). However, because we are searching for distances of at most $\delta$, it suffices for each $p \in L'$ to check all points $\{q | q \in R', p.y - \delta \leq q.y \leq p.y + \delta\}$. 
 
-It can be shown that for each point $p$, there is $\mathcal{O}(1)$ points that satisfy this property. Because each point in $R'$ is at least $\delta_r \geq \delta$ apart, arranging the points in the worst case would result in 6 points in the corners and sides of the bounding rectangle.
+It can be shown that for each point $p$, there is a constant number of points that satisfy this property. Because each point in $R'$ is at least $\delta_r \geq \delta$ apart, arranging the points in the worst case would result in 6 points in the corners and sides of the bounding rectangle.
```

Comment:
ok why not just directly state the number of points
or is it dependent on the input size idk

Reply:
<= 6 points

Reply:
I say that in the next sentence